distributions/demo-server/src/main/resources/config/default_retry.yaml

```diff
@@ -0,0 +1,14 @@
+
+  strategyMappings:
+  - regexes:
+    - "status code: 4\\d\\d"
+    strategy: !<Fatal> {}
+  - regexes:
+    - "simple"
+    strategy: !<Regular>
+      maxAttempts: 5
+      intervalMillis: 100
+  defaultRetryStrategy: !<Exponential>
+    maxAttempts: 5
+    initialIntervalMillis: 50
+    multiplier: 1.5
```

Comment:
When you document retry strategies, can you add that this is (currently) defined per-distribution? And that we are considering also defining per integration (transfer extension) to allow services to provide default behavior for retrying calling their service.

Reply:
Done.